packages/core/src/runtime.ts

```diff
@@ -569,7 +569,12 @@ export class AgentRuntime implements IAgentRuntime {
      * @param didRespond Whether the agent responded to the message.
      * @returns The results of the evaluation.
```

Comment:
didn't update the jsdoc block with new parameters so the docs are also up to date

Reply:
fixed